plaidml/bridge/openvino/ops/round.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset5.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+void registerRound() {
+  registerOp("Round", [](const Context& ctx) {
+    IE_ASSERT(ctx.operands.size() == 1);
+    auto I = ctx.operands.at(0);
+    auto* layer = ngraph::as_type<ngraph::opset5::Round>(ctx.layer);
+    auto roundMode = layer->get_mode();
+
+    switch (roundMode) {
+      case ngraph::op::v5::Round::RoundMode::HALF_TO_EVEN: {
+        auto unsignedI = op::abs(I);
+        auto intPart = edsl::cast(unsignedI, DType::INT32);
```

Comment:
What happens if this overflows I32? Maybe this is a moot point because FP32 can't represent every integer beyond (IIRC) 2^24, so rounding some float in the vicinity of 2^50 is going to be a no-op and the rounding mode doesn't matter. But what about an FP64 case?

This might just mean we want a round mode on edsl::round in the long run. In the short term though, perhaps we need to determine the type of the cast by the type of the input so that FP64 gets INT64 for the intPart